src/components/ik-controller.js

```diff
@@ -245,8 +245,8 @@ AFRAME.registerComponent("ik-controller", {
       this.ikRoot.el.components["player-info"].data.avatarType === AVATAR_TYPES.LEGACY
         ? LEGACY_HAND_ROTATIONS
         : HAND_ROTATIONS;
-    this.updateHand(handRotations.left, leftHand, leftController.object3D, true, this.isInView);
-    this.updateHand(handRotations.right, rightHand, rightController.object3D, false, this.isInView);
+    if (leftHand) this.updateHand(handRotations.left, leftHand, leftController.object3D, true, this.isInView);
```

Comment:
Hmm forgot that a custom avatar means they aren't going to get hands. Wonder if we should create some default hands, at least locally.

Reply:
Oh, yeah, good point. Not sure if I want to tackle it in this PR though.